mcp/runtime/src/main/java/io/quarkiverse/langchain4j/mcp/runtime/http/QuarkusHttpMcpTransport.java

```diff
@@ -84,6 +84,11 @@ public CompletableFuture<JsonNode> initialize(McpInitializeRequest request) {
                 .subscribeAsCompletionStage();
     }
 
+    @Override
+    public void checkHealth() {
+        // no transport-specific checks right now
```

Comment:
Just FYI, this will perform a `ping` operation and throw an exception if the server fails to respond in time. But don't worry about this, I'm going to do it (see https://github.com/quarkiverse/quarkus-langchain4j/issues/1414#issuecomment-2800847769)

Reply:
Ok, thanks for comment, let me know if I should do something about this.